ansible/inventory.yml

```diff
@@ -89,6 +89,9 @@ hosts:
       - spearhead:
           freebsd12-x64-1: {ip: 185.131.222.224}
 
+      - loongson:
+          Loongnix20-loongarch64-1: {ip: 10.40.64.29, user: loongson}
```

Comment:
I see the style of os name is lowercase. How about `loongnix20-loongarch64-1`?

Reply:
Sorry, my fault.